cfy_manager/utils/common.py

```diff
@@ -161,3 +164,47 @@ def manager_using_db_cluster():
         )
         and config[POSTGRESQL_SERVER]['cluster']['nodes']
     )
+
+
+def get_haproxy_servers(logger):
+    # Get the haproxy status data
+    try:
+        haproxy_csv = requests.get(
+            'http://localhost:7000/admin?stats;csv;norefresh'
+        ).text
+    except requests.ConnectionError as err:
+        logger.info(
+            'Could not connect to DB proxy ({err}), '
+            'retrying...'.format(err=err)
```

Comment:
this line is fake news
(because the retry is a level above)

Reply:
i'd probably just move the try/except a level above, because if this returns None, then the new db-list commands can't do anything anyway

Reply:
or move the retrying here

Reply:
Good point, I've made the retrying be logged elsewhere (since I already moved the retry out of here)